docs/components/input/demo/base-demo.md

```diff
@@ -1,5 +1,7 @@
 ```hbs template
-<Form::Controls::Input />
+<div class='w-96'>
+  <Form::Controls::Input />
+</div>
```

Comment:
Since these are now full-width by default, adding the container with the fixed width so they don't look ridiculous 😂